src/api/api.js

```diff
@@ -4,13 +4,15 @@ const ncNewsApi = axios.create({
   baseURL: "https://reddit-backend-client.onrender.com/api/",
 });
 
-export const getAllArticles = () => {
-  return ncNewsApi.get("/articles").then(({ data }) => {
-    data.articles.sort(
-      (a, b) => new Date(b.created_at) - new Date(a.created_at)
-    );
-    return data;
-  });
+export const getAllArticles = (topic) => {
+  return ncNewsApi
+    .get("/articles", { params: { topic: topic } })
+    .then(({ data }) => {
+      data.articles.sort(
```

Comment:
This sorting should have been implemented in your back end.

Reply:
It is, so I don't know why I put that there.  Have tested just to make sure, so will delete it and merge